versioned_docs/version-v2/config/01-overview.md

```diff
@@ -88,7 +88,7 @@ export default function() { // or export default async function()
 ```
 
 :::note
-The exported function can also be `async`.
```

Comment:
Can we back out the v2 changes? We're no longer maintaining this version of Stencil/its docs